include/api.php

```diff
@@ -560,6 +560,80 @@ function pll_count_posts( $lang, $args = array() ) {
+ * @param string              $taxonomy The taxonomy to which to add the term.
+ * @param PLL_Language|string $language The term language object or slug.
+ * @param array               $args {
+ *            Optional. Array of arguments for inserting a term.
+ *
+ *     @type string   $alias_of     Slug of the term to make this term an alias of.
+ *                                  Default empty string. Accepts a term slug.
+ *     @type string   $description  The term description. Default empty string.
+ *     @type int      $parent       The id of the parent term. Default 0.
+ *     @type string   $slug         The term slug to use. Default empty string.
+ *     @type string[] $translations The translation group to assign to the term with language slug as keys and `term_id` as values.
+ * }
+ * @return array|WP_Error {
+ *     An array of the new term data, `WP_Error` otherwise.
+ *
+ *     @type int        $term_id          The new term ID.
+ *     @type int|string $term_taxonomy_id The new term taxonomy ID. Can be a numeric string.
+ * }
+ */
+function pll_insert_term( string $term, string $taxonomy, $language, array $args = array() ) {
+	$language = PLL()->model->get_language( $language );
+
+	if ( ! $language instanceof PLL_Language ) {
+		return new WP_Error( 'invalid_language', __( 'Please provide a valid language.', 'polylang' ) );
+	}
+
+	return PLL()->model->term->insert( $term, $taxonomy, $language, $args );
+}
+
+/**
+ * Wraps `wp_update_term` with language feature.
+ *
+ * @since 3.7
+ *
+ * @param int   $term_id The ID of the term.
+ * @param array $args {
+ *     Optional. Array of arguments for updating a term.
+ *
+ *     @type string              $alias_of     Slug of the term to make this term an alias of.
+ *                                             Default empty string. Accepts a term slug.
+ *     @type string              $description  The term description. Default empty string.
+ *     @type int                 $parent       The id of the parent term. Default 0.
+ *     @type string              $slug         The term slug to use. Default empty string.
+ *     @type PLL_Language|string $lang         The term language object or slug.
+ *     @type string[]            $translations The translation group to assign to the term with language slug as keys and `term_id` as values.
+ * }
+ * @return array|WP_Error {
+ *     An array of the new term data, `WP_Error` otherwise.
+ *
+ *     @type int        $term_id          The new term ID.
+ *     @type int|string $term_taxonomy_id The new term taxonomy ID. Can be a numeric string.
+ * }
+ */
+function pll_update_term( int $term_id, array $args = array() ) {
+	$language = isset( $args['lang'] ) ? PLL()->model->get_language( $args['lang'] ) : null;
+
+	if ( false === $language ) {
+		return new WP_Error( 'invalid_language', __( 'Please provide a valid language.', 'polylang' ) );
+	}
+
+	if ( $language instanceof PLL_Language ) {
+		$args['lang'] = $language;
+	}
+
```

Comment:
Not need if done in the wrapped method.
```suggestion
```

Reply:
Removed in b54df4000a24ad2807375b7fbb6d623ea1dfe64c.